nats_events/nats_events/v1_0/nats_queue/config.py

```diff
@@ -0,0 +1,164 @@
+"""NATS Queue configuration."""
+
+import logging
+from typing import Any, Mapping, Optional
+
+from pydantic import BaseModel
+
+LOGGER = logging.getLogger(__name__)
+
+EVENT_TOPIC_MAP = {
+    "^acapy::webhook::(.*)$": "acapy-webhook-$wallet_id",

```

Comment:
Not sure if I have the right idea/location here for this.
I'd suggest that the target NATS subject uses `.` separator so that we can leverage some of the built in subject addressing capabilities in NATS. e.g. ` "^acapy::webhook::(.*)$": "acapy.webhook.$wallet_id",`